server/conf/cms/secrets.sample.py

```diff
@@ -1,31 +1,250 @@
-_SECRET_KEY = 'change_me'
-_DEBUG = True
-_ALLOWED_HOSTS = ['*']
+# TACC WMA SAD CMS SITE: FRONTERA-WEB.TACC.UTEXAS.EDU
+# CUSTOM SETTINGS VALUES.
+
+########################
+# DJANGO SETTINGS
+########################
+
+_SECRET_KEY = 'replacethiswithareallysecureandcomplexsecretkeystring'
+_DEBUG = True       # False for Prod.
+_CONSOLE_LOG_ENABLED = False    # Boolean check to turn on/off console logging statements.
+
+# Specify allowed hosts or use an asterisk to allow any host and simplify the config.
+# _ALLOWED_HOSTS = ['hostname.tacc.utexas.edu', 'host.ip.v4.address', '0.0.0.0', 'localhost', '127.0.0.1']   # In production.
+_ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost', '*']   # In development.
+
+# Boolean check to see if ldap is being used by the site.
+# Ensure the django-auth-ldap==2.0.0 package is uncommented
+# in the requirements.txt file or installed if using ldap.
+_LDAP_ENABLED = True
+
+# Boolean check to determine the appropriate database settings when using containers.
+_USING_CONTAINERS = True
+
+########################
+# DATABASE SETTINGS
+########################
+
+if _USING_CONTAINERS:
+    # used in container deployments.
+    _DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
+            'PORT': '5432',
+            'NAME': 'taccsite',
+            'USER': 'postgresadmin',
+            'PASSWORD': 'taccforever', # Change before live deployment.
+            'HOST': 'frontera_cms_postgres'
+        }
+    }
+else:
+    # used in local dev, venv or manual deployments.
+    _DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.postgresql_psycopg2',
+            'PORT': '5432',
+            'NAME': 'taccsite',
+            'USER': 'postgresadmin',
+            'PASSWORD': 'taccforever', # Change before live deployment.
+            'HOST': 'localhost'
+        }
+    }
+
+########################
+# DJANGO CMS SETTINGS
+########################
+
+# CMS Site (allows for multiple sites on a single CMS)
 _SITE_ID = 1
 _CMS_TEMPLATES = (
-    ('base.html', 'Base'),
-    ('cms_menu.html', 'CMS Menu'),
-    ('fullwidth.html', 'Fullwidth'),
-    ('sidebar_left.html', 'Sidebar Left'),
-    ('sidebar_right.html', 'Sidebar Right')
+    # Customize this
+    # FAQ: First template is default template
+    # REF: http://docs.django-cms.org/en/latest/how_to/install.html#templates
+    ('frontera-cms/templates/fullwidth.html', 'Fullwidth'),
+    ('fullwidth.html', 'DEPRECATED Fullwidth'),
 )
 
-# As needed:
-_CACHES = {
-    'default': {
-        'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-        'LOCATION': 'frontera_prtl_memcached:11211',
-    }
+########################
+# GOOGLE ANALYTICS
+########################
+
+# To use during dev, Tracking Protection in browser needs to be turned OFF.
+_GOOGLE_ANALYTICS_PROPERTY_ID = "UA-125525035-13"
```

Comment:
Is this a real property ID?

Reply:
Yeah, this should be something like this:

_GOOGLE_ANALYTICS_PROPERTY_ID = "UA-123ABC@%$&-#"

Reply:
Yeah, this should be something like this:

_GOOGLE_ANALYTICS_PROPERTY_ID = "UA-123ABC@%$&-#"

Reply:
Yes, it is a real property ID. But, it is **not** secret.

It can be seen, now, on Texascale, in the production markup as:
```
<script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-125525035-18"></script>
```

It can be seen, now, in Portal production markup as:
```
<script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-125525035-13"></script>
```

I will make the change, but please explain the reason for hiding this value.

Reply:
We aren't hiding the value, we are ensuring we dont generate unintended metric data for a portal that is not our property. Any portal deployed with this tracking ID in it could potentially generate metric data into google analytic account and pollute the stream. Since this is an example secrets file, and this dummy code works as a placeholder for the real ID that anyone deploying a portal would need to go acquire as part of the deployment setup process (documented) to place into their actual secrets file, there is no need for this file to expose one of our actual portal property IDs.

Reply:
Yup 👍. I just figured it out. Thank you. I am making the change now, and referencing this.

Reply:
Done: https://github.com/TACC/Frontera-Portal/pull/60/commits/8fe6e42031505fa29a4e7778bb8286ce7f3712b1